### ISSUE TITLE
Add documentation on how to update repo

### ISSUE THREAD

**Issue by katharinebreeze** [moj-analytical-services/writing_functions_in_r#55]
Be good to document:

* how to update read me, slides, example_code
* how to hide solutions
* any other tricks to be aware of?

To help others know how to make changes to the repo in future.

For how to update read me, could say something like:

> The read me, slides & example_code files in this repo are auto-generated from Rmd files in the `rmd_files` folder and should not be edited manually. 
> 
> To update the repo:
> 1.  Make branch
> 2. Make changes to `*.Rmd` files in the `rmd_files` folder
> 3. Update auto-generated files by running `render_rmarkdown_files.R` (but don't commit the auto-generated files)
> 4. Raise pull request & merge changes into master
> 5. Run `render_rmarkdown_files.R` on the master branch to update auto-generated files
> 
> The reason auto-generated files are only updated on the master branch is to avoid merge conflicts.

**Comment by chloepugh** [comment]
Yep that sounds like a good idea. For point 4. we might want to encourage anyone making changes to find someone to check the PR.

There's a decision to be made about where to store the documentation. Either we could add a section to the end of the main readme, or maybe a note could be adding to the preamble to direct presenters to a separate file with the instructions (or something else that I've not thought of!)

Also worth pointing out somewhere that most of the course content is stored in content.Rmd.

**Comment by katharinebreeze** [comment]
I've added something about reviewing PRs and explaining content.Rmd.

R.e. where to store the documentation - I like the idea of linking from the read me to another file.
